README.md

```diff
@@ -728,6 +728,83 @@ _You can enable the following settings in Xcode by running [this script](resourc
   let dictionary: [String: Any] = [:]
   ```
 
+* <a id='single-line-statement-braces'></a>(<a href='#single-line-statement-braces'>link</a>) The opening brace following a single-line statement should be on the same line as the rest of the statement. [![SwiftFormat: braces](https://img.shields.io/badge/SwiftFormat-braces-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#braces)
+
+  <details>
+
+  ```swift
+  // WRONG
+  if !planet.isHabitable
+  {
+    planet.terraform()
+  }
+
+  class Planet
+  {
+    func terraform()
+    {
+      generateAtmosphere()
+      generateOceans()
+    }
+  }
+
+  // RIGHT
+  if !planet.isHabitable {
+    planet.terraform()
+  }
+
+  class Planet {
+    func terraform() {
+      generateAtmosphere()
+      generateOceans()
```

Comment:
```suggestion
      generateOceans()
      pollutePlanet()
      findNewPlanet()
```

Reply:
![](https://media.giphy.com/media/l396Uasr95XqhSFJm/giphy.gif)

Reply:
seems wasteful to terraform, pollute, and then discard a planet all in a single run loop cycle -- maybe we'd want to `await pollutePlanet()` 🤪